### PR TITLE
{account} Bump up `azure-mgmt-subscription` SDK to `3.0.0`

### PR DESCRIPTION
**Pull request by PARADISSEEKR**
---

Azure-cli request for a Subscription API new version https://github.com/Azure/azure-cli/issues/20420
close https://github.com/Azure/azure-cli/issues/20420

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
account

File: src/account/azext_account/__init__.py [hunk 1]
```diff
@@ -19,8 +19,7 @@ def __init__(self, cli_ctx=None):
         account_custom = CliCommandType(
             operations_tmpl='azext_account.custom#{}',
             client_factory=cf_account)
-        super(SubscriptionClientCommandsLoader, self).__init__(cli_ctx=cli_ctx,
-                                                               custom_command_type=account_custom)
+        super().__init__(cli_ctx=cli_ctx, custom_command_type=account_custom)
```
Comment (zhoxing-ms) [inline src/account/azext_account/__init__.py:22]: Why do we need to modify here?

File: src/account/azext_account/manual/custom.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+def account_alias_create(client,
+                         alias_name,
+                         workload=None,
+                         billing_scope=None,
+                         display_name=None,
+                         subscription_id=None,
+                         reseller_id=None,
+                         no_wait=False):
+    body = {'properties': {'display_name': display_name, 'workload': workload, 'billing_scope': billing_scope,
+                           'subscription_id': subscription_id, 'reseller_id': reseller_id}}
```
Comment (zhoxing-ms) [inline src/account/azext_account/manual/custom.py:29]: I personally recommend transforming it into a structured format, which is more convenient for us to understand its structure
```Python
    body = {
            'properties': {
                'display_name': display_name, 
                'workload': workload, 
                'billing_scope': billing_scope,
                'subscription_id': subscription_id, 
                'reseller_id': reseller_id
            }
        }
```

File: src/account/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.2
++++++
+* Update SDK
```
Comment (zhoxing-ms) [inline src/account/HISTORY.rst:8]:
```suggestion
* Update SDK azure-mgmt-subscription version to 3.0.0 and api-version to 2021-10-01
```

File: src/account/azext_account/__init__.py (hunk 1 above)
Comment (PARADISSEEKR) [reply src/account/azext_account/__init__.py:22]: Syntax required in `azdev style`: `Consider using Python 3 style super() without arguments (super-with-arguments)`.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**